conda_lockfiles/dumpers/rattler_lock_v6.py

```diff
@@ -50,29 +57,35 @@ def _record_to_rattler_lock_v6(record: PackageRecord) -> dict[str, Any]:
     return package
 
 
-def export_to_rattler_lock_v6(prefix: str, lockfile_path: str | None) -> None:
-    prefix_data = PrefixData(prefix)
-    unique_channels = {(record.channel) for record in prefix_data.iter_records()}
-    if None in unique_channels:
-        raise EnvironmentExportNotSupported(LOCKFILE_FORMAT)
-    channels = sorted([{"url": str(url)} for url in unique_channels])
-    packages = sorted(
-        [_record_to_rattler_lock_v6(record) for record in prefix_data.iter_records()],
-        key=lambda x: x["conda"],
-    )
-    env_subdir_pkgs = [{"conda": package["conda"]} for package in packages]
-    environments = {
-        "default": {
-            "channels": channels,
-            "packages": {
-                context.subdir: env_subdir_pkgs,
-            },
-        }
-    }
-    output = {
+def _to_dict(env: Environment) -> dict[str, Any]:
+    missing_urls = [package for package in env.explicit_packages if package.url is None]
+    if missing_urls:
+        raise EnvironmentExportNotSupported(
+            FORMAT,
+            f"The following packages have no URL: {dashlist(missing_urls)}",
+        )
```

Comment:
This is done identically in `conda-lock-v1`, should it be a helper function? Or performed elsewhere?